docs/src/documentation/05-development/04-migration-guides/01-v23.mdx

```diff
@@ -48,6 +48,26 @@ intl.formatMessage({
 
 Feel free to customize the translations according to your needs, by providing more specific context to the object it refers to.
 
+### Prop name change in Stepper and StepperStateless
+
+The prop name has been changed from `ariaLabelledBy` to `ariaLabelledby` in the Stepper and StepperStateless components to align with the correct ARIA attribute naming convention.
+
+**Before:**
+
+```jsx
+<Stepper ariaLabelledBy="label-id" />
+<StepperStateless ariaLabelledBy="label-id" />
+```
+
+**Now:**
+
+```jsx
+<Stepper ariaLabelledby="label-id" />
+<StepperStateless ariaLabelledby="label-id" />
+```
+
+Feel free to use the codemod to update the prop name in your code.
```

Comment:
The codemod instructions point to a different file.

It should be only one file with all the transforms